tools/packaging/kata-deploy/local-build/dockerbuild/install_yq.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# Copyright (c) 2018-2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+yq_path="/usr/local/bin/yq"
+yq_pkg="github.com/mikefarah/yq"
+goos="linux"
+case "$(uname -m)" in
+	aarch64) goarch="arm64";;
+	ppc64le) goarch="ppc64le";;
+	x86_64) goarch="amd64";;
+	s390x) goarch="s390x";;
+	*) echo "unsupported architecture: $(uname -m)"; exit 1;;
+esac
+yq_version=3.4.1
```

Comment:
Might be worth adding a comment explaining why this version is being used.

Reply:
Hmm. I guess because this is the same version used [here](https://github.com/kata-containers/tests/blob/d87e9cf8f254a08be868c55a195ce3c300164658/.ci/install_yq.sh#L28).

Although one would think it deserves an issue to track the yq version as part of the versions.yaml file from the kata-containers repo, we need yq installed to easily read the versions.yaml file. :-)

Reply:
@jcvenegas, couldn't we re-use the whole file from https://github.com/kata-containers/tests/blob/main/.ci/install_yq.sh#L28 ?

Reply:
@fidencio @jodh-intel  yes, it is the same version we use in tests repo, I have copied the script as I dont want to depend on tests repository to build  kata, (avoid double depends on  or sync repositories , but what I can do to have more organized code is to move them under ci  directory.

Reply:
I agree, it may be nice to defined a yq version somewhere,  but we need to yq  to read the yaml xD, we can have a comment inside that define what is the recommend yq version we use. so the yq installer reads it without use yq.